client/landing/stepper/declarative-flow/internals/steps-repository/pattern-assembler/utils.ts

```diff
@@ -36,14 +36,17 @@ export const handleKeyboard =
 export function createCustomHomeTemplateContent(
 	stylesheet: string,
 	hasHeader: boolean,
-	hasFooter: boolean
+	hasFooter: boolean,
+	hasSections: boolean
 ) {
 	const content: string[] = [];
-
+	const noSelection = ! hasHeader && ! hasFooter && ! hasSections;
```

Comment:
This is not required. We could insert a placeholder when the header/footer is not selected. What do you think?

Reply:
I think this is required because if the user adds a section, we don't need to add the header and footer template parts. The goal is to avoid a blank screen without any pattern or template part in the editor.

Reply:
In other words, we always don't need to add the header and footer template parts. Users can choose if they want a header or footer in the PA. I think the goal is not to add a new feature to show always the header and footer template parts in the editor. What do you think?

Reply:
As mentioned in p1667783724157909/1667457518.728119-slack-CRWCHQGUB, it's a great educational opportunity. Additionally, the placeholder won't be shown when you preview the site, so it might be better to have it to avoid people not knowing how to insert a header/footer. But I'm okay with both, and we could address this after the CfT 🙂